### PR TITLE
Added  of Dynamic List and Table Operations

### PR DESCRIPTION
**Pull request by nikvpatil97**
1.Render the list from array of fruit string.
2.Render the list from array of student name string.
3.Render the table from array of object.

### CODE REVIEW

File: DOM/dynamicListAndTable/fruitList.html [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+  <link href="fruit_list.css" rel="stylesheet" />
+</head>
+
+<body>
+
+  <div class="container">
+    <p>Fruits</p>
+    <div>
+      <ol id="fruitList" class="fruit_list">
+        <!-- <li>Apple</li>
```
Comment (ajaybirari1993) [inline DOM/dynamicListAndTable/fruitList.html]: Please remove cemented code.

File: DOM/dynamicListAndTable/fruit_list.css [hunk 2]
```diff
@@ -0,0 +1,11 @@
+.container {
+  display: flex;
+  padding: 1rem;
+  /* justify-content: center;
```
Comment (ajaybirari1993) [inline DOM/dynamicListAndTable/fruit_list.css]: Same as above.

File: DOM/dynamicListAndTable/fruitList.html (hunk 1 above)
Comment (nikvpatil97) [reply DOM/dynamicListAndTable/fruitList.html]: I was removed the commented part.

File: DOM/dynamicListAndTable/fruit_list.css (hunk 2 above)
Comment (nikvpatil97) [reply DOM/dynamicListAndTable/fruit_list.css]: I was removed the commented part.

**Review by ajaybirari1993 — APPROVED** [review]

**Pull request merged**